test/unit/runIntersectionsTests.js

```diff
@@ -0,0 +1,18 @@
+var tape = require('tape'),
+    diff = require('difflet')({ indent : 2, comment : true });
+
+var common = {
+    // a visual deep diff rendered using console.error()
+    diff: function( actual, expected ){
+        console.error( diff.compare( actual, expected ) );
+    }
+};
+
+var tests = [
+    require('./helper/intersectionsParsing'),
+    require('./controller/predicates/is_intersection_layer')
```

Comment:
There's no reason for these tests to be separate from our main tests (I'm sure it was easier initially when testing, so no worries). Please move these two lines into `test/unit/run.js` and remove this file.

Reply:
@orangejulius Taken care of!